css/components/_navigation.scss

```diff
@@ -1,3 +1,73 @@
-.nav-item {
-  font-family: $font-header;
+@mixin centerer {
```

Comment:
more descriptive names for mixins are helper, I didn't expect anyone to use them actually! hah, normally we keep these in their own sass file